app/service/rest.py

```diff
@@ -700,7 +713,9 @@ def get_single_month_notification_stats_for_service(service_id):
     month_year = datetime(year, month, 10, 00, 00, 00)
     start_date, end_date = get_month_start_and_end_date_in_utc(month_year)
 
-    results = dao_fetch_stats_for_service_from_days(service_id, start_date, end_date)
+    __, results = dao_fetch_stats_for_service_from_days(
```

Comment:
Just double checking that we do need actually need the first value returned here, only `results`, right?

Reply:
Confirming that this is intended and the first value is not needed.